config/webpacker.yml

```diff
@@ -34,6 +34,9 @@ default: &default
     - .woff2
 
   extensions:
+    - .tsx
+    - .ts
+    - .jsx
```

Comment:
should we leave out `jsx` so we force react to be typescript only?  are we going to use `jsx` anywhere else?

Reply:
Yes, good point!